### PR TITLE
DateTimePicker: Fix for IE

### PR DESCRIPTION
**Pull request by jmophoto**
Two IE issues:

1. `String.includes` is not supported in IE. Changed that to `String.indexOf`
2. The onDateSelect was not triggered when the calendar was added to the page. Changed to the pattern used in the DatePicker with `display` hide/show.

### CODE REVIEW

File: src/components/DateTimePicker.js [hunk 1]
```diff
@@ -114,13 +114,13 @@ const DatePicker = React.createClass({
...
-    const am = value.includes('am');
-    const pm = value.includes('pm');
+    const am = value.indexOf('am');
+    const pm = value.indexOf('pm');
     const minute = Number(time[1].substring(0, 2));
     let hour = Number(time[0]);
 
-    hour = am && hour === 12 ? 0 : hour;
-    hour = pm && hour !== 12 ? hour + 12 : hour;
+    hour = am >= 0 && hour === 12 ? 0 : hour;
```
Comment (jtanner) [inline src/components/DateTimePicker.js]: I think `am` and `pm` should remain booleans, can you move `>= 0` up to the `const` declarations?

Comment (cerinman) [inline src/components/DateTimePicker.js]: I like that idea.

```javascript
const am = value.includes('am') >= 0;

hour = am && hour === 12 ? 0 : hour;
```

**Comment by jtanner** [comment]
:shipit:

File: src/components/DateTimePicker.js (hunk 1 above)
Comment (jmophoto) [inline src/components/DateTimePicker.js]: Agreed. Changed that and pushed.

**Pull request merged**